src/modules/RA/DataSource/components/DataSourceCSVFields.js

```diff
@@ -0,0 +1,193 @@
+
+const DataSourceCSVFields = ({ translate: t, type, ...props }) => (
+  <FieldGroup {...props}>
+    <FileInput
+      source="file"
+      label="datasource.form.file.related-files"
+      multiple={false}
+      placeholder={t('datasource.form.file.placeholderCSV')}
+    >
+      <FileField source="file_data" title="title" />
+    </FileInput>
+
+    <SelectInput
+      source="encoding_field"
+      label="datasource.form.optionsCSV.encodingField"
+      validate={[required()]}
+      choices={fieldEncodingChoices}
+      format={v => `${v}`}
+      parse={v => v}
+    />
+
+    <RadioButtonGroupInput
+      source="coordinates_field"
+      label="datasource.form.coordinates.coordinatesField"
+      choices={[
+        { id: 'one_column', name: t('datasource.form.coordinates.onecolumnField') },
+        { id: 'two_columns', name: t('datasource.form.coordinates.twocolumnsField') },
+      ]}
+    />
+
+    <FormDataConsumer>
+      {({ formData }) => {
+        if (formData.coordinates_field && formData.coordinates_field === 'one_column') {
+          return (
+            <>
+              <RadioButtonGroupInput
+                source="order_coordinates_field"
+                label="datasource.form.coordinates.orderCoordinatesField"
+                choices={[
+                  { id: 'xy', name: 'X / Y' },
+                  { id: 'yx', name: 'Y / X' },
+                ]}
+              />
+              <TextInput
+                source="latlong_field"
+                type="text"
+                label="datasource.form.coordinates.latlongField"
+              />
+              <SelectInput
+                source="F"
+                label="datasource.form.coordinates.separatorCoordinatesField"
+                defaultValue="comma"
+                validate={[required()]}
+                choices={[
+                  { id: 'comma', name: t('datasource.form.optionsCSV.separator.comma') },
+                  { id: 'semicolon', name: t('datasource.form.optionsCSV.separator.semicolon') },
+                  { id: 'space', name: t('datasource.form.optionsCSV.separator.space') },
+                ]}
+                format={v => `${v}`}
+                parse={v => v}
+              />
+            </>
+          );
+        }
+      }}
+    </FormDataConsumer>
+
+    <FormDataConsumer>
+      {({ formData }) => {
+        if (formData.coordinates_field && formData.coordinates_field === 'two_columns') {
+          return (
+            <>
+              <TextInput
+                source="latitude_field"
+                type="text"
+                label="datasource.form.coordinates.latitudeField"
+              />
+              <TextInput
+                source="longitude_field"
+                type="text"
+                label="datasource.form.coordinates.longitudeField"
+              />
+            </>
+          );
+        }
+      }}
+    </FormDataConsumer>
+
+    <SelectInput
+      source="scr_field"
```

Comment:
`scr`

Reply:
It has been change to `coordinates_representation_system`